material-github.user.css

```diff
@@ -1334,7 +1337,8 @@
 		color: var(--material-code-yellow);
 	}
 	.highlight .nf,
-	.highlight .nb{
```

Comment:
Shouldn't `.nb` be red ?

Reply:
I was trying to improve the colors of this latex code in the [markdown notebook](https://github.com/jupyter/notebook/blob/master/docs/source/examples/Notebook/Working%20With%20Markdown%20Cells.ipynb)
![image](https://user-images.githubusercontent.com/34294427/101121956-ea719680-3623-11eb-864a-ed56424b978a.png)
If it conflicts with your color convention, I can change it back.

Reply:
Yes, it would affect all languages so I prefer you change it back.